game.js

```diff
@@ -36,6 +22,75 @@ const saveStates = (playerState, savedState) => {
   });
 };
 
+const createBoard = (size) => {
+const gameContainer =  document.querySelector('.game');
+gameContainer.classList.remove("hidden");
+if(game.boardSize !== "")
+gameContainer.classList.remove(game.boardSize);
```

Comment:
what is the purpose of this line? there is class with size? it doesn't make sense

Reply:
I just taking the class name of 'game-container' from boardSize(x3/x5)